app/views/assessor_interface/verify_references/index.html.erb

```diff
@@ -0,0 +1,64 @@
+<% content_for :page_title, "#{"Error: " if @form.errors.any?}Work references" %>
+<% content_for :back_link_url, assessor_interface_application_form_path(@form.application_form) %>
+
+<%= render "shared/assessor_header", title: "Work references", application_form: @view_object.application_form %>
+
+<p>This page shows all the work references requested for this application and the number of months of work experience that each one represents.</p>
+<p>You can select and review any reference that has the status <span>RECEIVED</span>.</p>
+
+<ol class="app-task-list">
+  <li>
+    <h2 class="app-task-list__section">
+      Work references
+    </h2>
+    <ul class="app-task-list__items govuk-!-padding-left-0">
+      <% @view_object.reference_requests.each_with_index do |reference_request, idx| %>
+        <li class="app-task-list__item">
+          <span class="app-task-list__task-name">
+            <%= govuk_link_to(
+              @view_object.name_and_duration(reference_request.work_history, most_recent: idx.zero?),
+              [:edit, :assessor_interface, @form.application_form, @form.assessment, reference_request]
+            ) %>
+          </span>
+          <strong class="govuk-tag app-task-list__tag govuk-tag--green" id="reference-status"><%= reference_request.state %></strong>
+        </li>
+      <% end %>
+    </ul>
+  </li>
+</ol>
+<h2 class="govuk-heading-s">Check:</h2>
+<ul class="govuk-list govuk-list--bullet govuk-list--spaced">
+  <li>the reference from the current or most recent role has been received and is valid (if required)</li>
+  <li>the references that you’ve checked and marked as valid add up to the required number of months for this application to progress</li>
+</ul>
+
+<%= form_with(
+  model: @form,
+  url: [:assessor_interface, @form.application_form, @form.assessment, :update_verify_references],
+  method: :put
+) do |f| %>
+  <%= f.govuk_error_summary %>
```

Comment:
We haven't been consistent about this in the app, but I think we should move this to the top of the file so that the error message, if there is one, appears above all the other content.